shared/constants/fs.tsx

```diff
@@ -1046,6 +1046,12 @@ export const getTlfPath = (path: Types.Path): Types.Path | null => {
   return elems.length > 2 ? Types.pathConcat(Types.pathConcat(defaultPath, elems[1]), elems[2]) : null
 }
 
+export const hasPublicTag = (path: Types.Path): boolean => {
+  return (
+    Types.getPathVisibility(path) === Types.TlfType.Public && path !== Types.stringToPath('/keybase/public')
+  )
```

Comment:
This is in constants, so could probably be more efficient, e.g. checking for a prefix and making sure the length is larger than `/keybase/public`'s length?